packages/sdk/src/across/clients/bridgePool.ts

```diff
@@ -244,6 +269,11 @@ function joinPoolState(poolState: Awaited<ReturnType<PoolState["read"]>>): Pool
     exchangeRateCurrent: poolState.exchangeRateCurrent.toString(),
     exchangeRatePrevious: poolState.exchangeRatePrevious.toString(),
     estimatedApy,
+    estimatedApr,
+    estimatedApyBlocks,
+    estimatedAprBlocks,
```

Comment:
Why are we still computing/returning this? Can we drop it?

Reply:
yes i can remove